public/index.html

```diff
@@ -1,12 +1,21 @@
-<!DOCTYPE html>
+<!DOCTYPE html lang="en">
 <html>
-<head>
-  <title>Photo Gallery Proxy</title>>
-</head>>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Photo Gallery Proxy</title>
 
-  <div id="photoGallery">Hello</div>div>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet">
+  </head>
+  <body>
 
-  <script></script>
-</body>
-</html>>
+    <div id="photos-and-info"></div>
+    <div id="photo-gallery"></div>
```

Comment:
Do the reviews go before the photo-gallery or after?

Reply:
I think reviews come after photos, but on the airbnb site, they have other information breaking up your section from my section, so its not so photo heavy at the top of the page